src/main.ts

```diff
@@ -21,6 +21,8 @@ export async function main(): Promise<Outputs> {
     const deploymentMode = getInput('deploymentMode')
     const deploymentName = getInput('deploymentName')
     const parameters = getInput('parameters')
+    const managementGroupId = getInput('managementGroupId')
```

Comment:
does it need to be id or would name be okay?

Reply:
The CLI command explicitly specifies it requires the Id:
`--management-group-id -m           [Required] : The management group id to create deployment at.`